al4007_projects/hybriddatastructures.md

```diff
@@ -0,0 +1,10 @@
+# hybrid-data-structures
```

Comment:
move to `projects/cpp/hybriddatastructures.md`

Reply:
fixed in next commit